src/google-maps/map-directions-renderer/map-directions-service.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// Workaround for: https://github.com/bazelbuild/rules_nodejs/issues/1265
+/// <reference types="googlemaps" />
+
+import {Injectable, NgZone} from '@angular/core';
+import {Observable} from 'rxjs';
+
+export interface MapDirectionsResponse {
+  status: google.maps.DirectionsStatus;
+  result?: google.maps.DirectionsResult;
+}
+
+/**
+ * Angular service that wraps the Google Maps DirectionsService from the Google Maps JavaScript
+ * API.
+ *
+ * See developers.google.com/maps/documentation/javascript/reference/directions#DirectionsService
+ */
+@Injectable()
```

Comment:
Should this be `providedIn: 'root'`? That way it can be removed from the `providers` array and tree shaken when not in use.

Reply:
Done.